lang/da.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: en\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Anvendelse: vrms-rpm [options]"
+msgid "msg_help_option_ascii"    
+msgstr "    Vis rms ASCII-kunst når betalingssoftware er fundet,\n"
+       "    eller når betalingssoftware er  10%% eller mere af total."
+msgid "msg_help_option_explain"
+msgstr "    Vis licenser i oversigten, for at fremhæve\n
+       "    gratis / betalings-klassificering."
+msgid "msg_help_option_help"
+msgstr "    Vis denne hjælp og exit."
+msgid "msg_help_option_list"
+msgstr "    Vis ikke kun antal gratis / betalings-pakker, men vis\n
+       "    også navn. Standardinstillingen er 'nonfree'."
+msgid "msg_help_option_version"
+msgstr "    Vis information om version og exit."
+msgid "msg_list_noargument"
+msgstr "%s: --vis påkrævede valgmuligheder for et argument"
+msgid "msg_list_badargument"
+msgstr "%s: argumentet til --list muligheden skal være én af de følgende 'none', 'free', 'nonfree' eller 'all'"
+msgid "msg_unknown_option"
+msgstr "%s: ukendt valgmulighed '%s'"
+msgid "msg_total_free"
+msgstr "%d gratis pakker"
+msgid "msg_total_nonfree"
+msgstr "%d betalings-pakker"
+msgid "msg_rms_happy_ascii"
+msgstr "Kun gratis pakker - rms ville være stolt!                      rms  er tilfreds."
```

Comment:
I'm just guessing here, but "gratis" commonly means "free" as in "no cost". Is this also the case in Danish? If so, this _needs_ to be changed. Free Software is not about the price, but user freedom. It's free as in "freedom", not as in "free beer".

Reply:
Yes, 'gratis' translates to free (as in payment-free) in danish, and you're talking about free (as in _open_ software). Wouldn't it be better to use the more precise expression of open / closed software in the EN translation to avoid confusion?
Every pending translation uses the free (as in payment-free) expression from what I understand, and that's clearly not what was intended.

Reply:
"Free Software" has been in use since the 80s, so unfortunately it's a bit too late to debate on whether "open" would be better.

I checked the other translations after spotting this issue in yours and, based on Wikipedia article titles, they're ok.